migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,82 @@
+:_content-type: ASSEMBLY
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
```

Comment:
This title is fine in the topic map, where we need a short title. However, I think that in the overview itself, a longer title would be clearer. I suggest: "Migration from OpenShift Container Platform 3 to 4 overview".